samples/bluetooth/peripheral_power_profiling/src/pwr_service.h

```diff
@@ -0,0 +1,72 @@
+/*
+ * Copyright (c) 2022 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+ */
+
+#ifndef PWR_SERVICE_H_
+#define PWR_SERVICE_H_
+
+#include <zephyr/bluetooth/conn.h>
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+/** @brief PWR Service event. */
+enum pwr_service_evt {
+	/** PWR Service notification enabled. */
+	PWR_SERVICE_EVT_NOTIFY_ENABLED,
+
+	/** PWR Service notification disabled. */
+	PWR_SERVICE_EVT_NOTIFY_DISABLED
+};
```

Comment:
We use this approach in other services to indicate notification state. I would use it here as well

Reply:
I see. Then, it is yet another item that should be aligned across our service modules. See the DDFS service as an example of my review suggestion:

https://github.com/nrfconnect/sdk-nrf/blob/main/include/bluetooth/services/ddfs.h#L120

@carlescufi, @grochu, what's your take on it? Two different ways of creating the callback API may be confusing for NCS users. All services should use the same approach for consistency.

Reply:
Maybe it would be good to prepare a common instruction how to design BLE services. At this moment we use mostly enumerator for indicating the CCC state. Personally I don't have anything against it

Reply:
Initially, DDFS used enum-based event. The enum structure consisted of 6 types of events. 
During the review process, this was changed to dedicated functions.

Reply:
I am not planning to block this PR because of this misalignment since the misalignment is already present in other service modules and the PWR service is not even a dedicated module but rather application code.

However, I believe that callback API for CCCD status indication is something that we should align in our service modules in the future.